pkg/mcs/scheduling/server/server.go

```diff
@@ -469,6 +469,12 @@ func (s *Server) startWatcher() (err error) {
 	return err
 }
 
+// GetOpts returns the persist config.
+// It's used to test.
+func (s *Server) GetOpts() *config.PersistConfig {
```

Comment:
How about using getcluster?

Reply:
It has not `SetPlacementRuleEnabled`

Reply:
Maybe you can add it?

Reply:
I will add it in another pr about refactor cluster

Reply:
Or rename it to GetPersistConfig